### PR TITLE
[FIX] l10n_dk: misconfigured tax

### PR DESCRIPTION
**Pull request by malb-odoo**
Before this commit the tax grid of the tax was wrong. The value of this tax
should be put in the total sales of goods and services in Field C in the tax
report.

link to documentation:
https://skat.dk/erhverv/moms/moms-ved-handel-med-udlandet/moms-ved-handel-med-virksomheder/moms-ved-handel-med-lande-uden-for-eu/moms-ved-salg-af-varer-og-ydelser-i-lande-uden-for-eu

task: 4132444

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by malb-odoo** [comment]
@kenneth-vkd Here is the pr :smile:

File: addons/l10n_dk/data/template/account.tax-dk.csv [hunk 1]
```diff
@@ -63,12 +63,12 @@
...
-"tax_s4",20,"S-%EU-V-0","Sales of goods B2B (% EU), 0% VAT","0 %","0.0","percent","sale","False","True","base","invoice","","","","Varesalg B2B (% EU), 0% moms","0 %"
-"","","","","","","","","","","tax","invoice","dk_coa_7740","+MVU","","",""
-"","","","","","","","","","","tax","invoice","dk_coa_7700","-KM",-100,"",""
-"","","","","","","","","","","base","refund","","","","",""
-"","","","","","","","","","","tax","refund","dk_coa_7740","-MVU","","",""
-"","","","","","","","","","","tax","refund","dk_coa_7700","+KM",-100,"",""
+"tax_s4",20,"S-%EU-V-0","Sales of goods B2B (% EU), 0% VAT","0 %","0.0","percent","sale","False","True","base","invoice","","R-C-MR","","Varesalg B2B (% EU), 0% moms","0 %"
+"","","","","","","","","","","tax","invoice","dk_coa_7740","","","",""
+"","","","","","","","","","","tax","invoice","dk_coa_7700","",-100,"",""
+"","","","","","","","","","","base","refund","","-R-D-MR","","",""
```
Comment (kenneth-vkd) [inline addons/l10n_dk/data/template/account.tax-dk.csv:69]: @malb-odoo Not entirely correct here. It should be `+R-C-UR` for `invoice` and `-R-C-UR` for `refund` as the tax code is for sales outside the EU

Comment (malb-odoo) [reply addons/l10n_dk/data/template/account.tax-dk.csv:69]: missed the copy paste :facepalm: ahaha

Comment (malb-odoo) [reply addons/l10n_dk/data/template/account.tax-dk.csv]: You have seen nothing :eyes: (just pushed the real fix ahah)

**Review by clbr-odoo — COMMENTED** [review]
Small nitpick, other than that it LGTM technically speaking :slightly_smiling_face: 
Can you just link the task (id=4132444) you created in the PR & commit message please ?
Maybe also link the legal ref in the commit message 
https://skat.dk/erhverv/moms/moms-ved-handel-med-udlandet/moms-ved-handel-med-virksomheder/moms-ved-handel-med-lande-uden-for-eu/moms-ved-salg-af-varer-og-ydelser-i-lande-uden-for-eu

**Comment by clbr-odoo** [comment]
@kenneth-vkd We will merge it soon in case you want to double-check again :slightly_smiling_face:

**Review by clbr-odoo — APPROVED** [review]
Thanks !
@robodoo r+

**Comment by kenneth-vkd** [comment]
@malb-odoo sorry to tag you on this closed PR, but it seems the migration script from this PR is causing some issues when an updated build is applied on databases that were upgraded from a previous version of Odoo.

We have observed two issues:
1. On very old databases (started prior to v13), all the tax codes are recreated
2. On databases that use `account_asset` from odoo/enterprise, the migration script raises a not-null constraint

Perhaps the migration script needs to be adjusted to account for databases initially started on older versions of Odoo, where the xmlid was not adjusted during upgrades to match with the corresponding xmlid of the new version

**Comment by malb-odoo** [comment]
@kenneth-vkd Hello ! I discussed a bit to see what we will do, here is the final response:

For the point 1: We don't plan to do anything for very old database, they should fix their config by themself, we want to encourage people to upgrade to not miss new functionality :smile: 

2: That not normal, it's possible that the script must be a end-migrate instead of a post. If you have some time you can try to change the script to an end migrate ? otherwise i will try to do it as soon as possible but with the oxp coming i will not have a lot of time :sweat_smile:

**Comment by kenneth-vkd** [comment]
@malb-odoo thank you for looking into this
We have noted the decision from Odoo regarding customers who have already upgraded from a previous version.

I will most likely not have the time to attempt moving the migration script from the `post` stage to the `end` stage as we have also discovered that the migration script is executed by the odoo upgrade platform and thereby affects any database migrated from fx. v16 where the corresponding tax code was correctly configured. So we will be working with the migration support on handling the issue for customers going through the upgrade going forward

**Pull request closed without merging**